src/components/ErrorBoundary/index.tsx

```diff
@@ -216,16 +217,15 @@ const updateServiceWorkerInBackground = async () => {
   }
 }
 
-export default function ErrorBoundary({ children }: PropsWithChildren): JSX.Element {
+const onError = async (error: Error) => {
+  logErrorWithNewRelic({ error })
+  updateServiceWorkerInBackground()
+}
+
+export default function TopLevelErrorBoundary({ children }: PropsWithChildren): JSX.Element {
   return (
-    <Sentry.ErrorBoundary
```

Comment:
We talked about using Sentry a few times in the past, I think @sudoFerraz and I agree that ultimately it would be good to have both NewRelic _and_ Sentry, the latter for more details about the user path leading to an error etc... To be discussed I guess?

Reply:
Fair enough, let's discuss that offline next week when Gabriel's back as well.